pom.xml

```diff
@@ -25,11 +25,10 @@
     <description>A project that demonstrates Hello World using HTML</description>
 
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.build.timestamp.format>yyyyMMdd'T'HHmmss</maven.build.timestamp.format>
+        <jakartaee-api.version>9.1.0</jakartaee-api.version>
```

Comment:
JBoss EAP 8 only supports Jakarta EE 10 - so this may not work as expected (it probably does for this simple app, but it'd be better to use the EE 10 dependencies)

Reply:
Yes, agreed. Use 10.0.0. See https://mvnrepository.com/artifact/jakarta.platform/jakarta.jakartaee-api/10.0.0

Reply:
Thank you @jamesfalkner